.github/workflows/build-and-test.yml

```diff
@@ -289,6 +289,13 @@ jobs:
         with:
           name: CoverageReport # Artifact name
           path: coveragereport # Directory containing files to upload
+      
+      - name: Upload coverage to Codecov
```

Comment:
Does this replace or complement the existing step that comments on PRs with code coverage?